doc/manual/upgrade.xml

```diff
@@ -129,7 +129,14 @@
           <listitem>
             <para>stores Mac Metadata (eg FinderInfo, AFP Flags, Comment,
             CNID) in an Extended Attributed named
-            “<filename>org.netatalk.Metadata</filename>”</para>
+            “<filename>org.netatalk.Metadata</filename>”<itemizedlist>
+                <listitem>
+                  <para>Additionally, on macOS hosts, FinderInfo is 
+                  natively stored in the file system and appears as an
+                  Extended Attribute named
+                  “<filename>com.apple.FinderInfo</filename>”</para>
+                </listitem>
+              </itemizedlist></para>
```

Comment:
I suggest we qualify this statement with something like "As of Netatalk 4.1.0..."